### PR TITLE
fix: Avoid non-POSIX readlink parameter

### LINKED ISSUE #103: Integration test error in OS X
The script that runs the integration is not compatible with OS X.
The tests fails on commands `readlink -f` and `cp -ar`, since those have different options interface  on that platform.

- readlink
fails with `readlink: illegal option -- f`

- `cp -ar` 
fails with `cp: the -R and -r options may not be specified together.`  The flag -a expands to -pPR in OS X, there the conflict.

### PR DESCRIPTION
**Pull request by badboy**
Fixes #103

### CODE REVIEW

**Comment by badboy** [comment]
cc @emig

File: tests/integration/run-locally.sh [hunk 1]
```diff
@@ -2,9 +2,10 @@
...
 
-cd $(dirname $(readlink -f $0))
+SCRIPTPATH=$(CDPATH= cd -- "$(dirname -- $0)" && pwd -P)
+cd "$SCRIPTPATH"
 
-export WORKSPACE=$(readlink -f ../../..)/workspace
+export WORKSPACE="$(CDPATH= cd ../../.. && pwd -P)/workspace"
 
 git clone https://github.com/sstephenson/bats/ || true
 export PATH=$(pwd)/bats/bin:$(readlink -f ../../target/debug):$PATH
```
Comment (emig) [inline tests/integration/run-locally.sh]: nice. Is it ok to still use `readlink -f` ?

Comment (badboy) [inline tests/integration/run-locally.sh]: D'oh!, that one needs to be fixed as well

**Comment by badboy** [comment]
@not-a-robot r+

**Pull request merged**